src/Main.hs

```diff
@@ -297,7 +297,7 @@ $excText1
 
 
 rebootCmd :: MonadIO io => Text -> io Turtle.ExitCode
-rebootCmd target = Turtle.shell [text|ssh $target sudo reboot|] empty
+rebootCmd target = Turtle.shell [text|ssh -o 'ServerAliveInterval=1' $target sudo reboot|] empty
```

Comment:
Is this long enough?  My understanding is that the default `ServerAliveCountMax` is 3, so wouldn't this only wait for 3 seconds before giving up?

Reply:
@Gabriel439 all i want is for the ssh session to exit when during reboot the interface is stopped before sshd,,,  i.e. i want ssh to fail as fast as possible~

i have a script to automate deployment to multiple appliances,,, due to the possible ssh hanging, my script hangs in the middle of deploy and it is really annoying

Reply:
Oooh, I see.  Alright, that makes sense to me!